config/database/init.go

```diff
@@ -6,21 +6,30 @@ import (
 	"github.com/expenseledger/web-service/config"
 )
 
-var (
+type configFields struct {
 	DBUser string
 	DBPswd string
 	DBName string
 	DBPort string
 	DBURL  string
-)
+}
+
+var configs configFields
 
 func init() {
-	if config.Mode == "PRODUCTION" {
-		DBURL = os.Getenv("DATABASE_URL")
+	gConfigs := config.GetConfigs()
+
+	if gConfigs.Mode == "PRODUCTION" {
+		configs.DBURL = os.Getenv("DATABASE_URL")
```

Comment:
Should we have db for dev ?

Reply:
You mean like a separate db? Yes, probably we should. I'm just gonna put that in to-do list, along with the need for staging branch. Right now the dev db is just your local db. Here: see the [README.md](https://github.com/expenseledger/web-service/blob/master/README.md) for the instructions to install the web service locally.